src/theme/index.js

```diff
@@ -0,0 +1,7 @@
+import { gradient } from "./gradient";
+import { palette } from "./palette";
+
+export const theme = {
```

Comment:
When building an app we usually have a "theme", a collection of colors, gradients, opacities, spacing units, etc that we want to use over and over. This is a pretty simple structure I put together here for use in JS -- but CSS now supports variables so we could move to CSS instead of JS.

https://medium.com/@brcsndr/you-dont-know-css-variables-theming-d7a5c142d6ef